src/nbsap/forms.py

```diff
@@ -739,6 +741,16 @@ class EuAichiStrategyForm(forms.Form, ChoicesMixin):
     other_eu_indicators = chosenforms.ChosenMultipleChoiceField(
         overlay="Select indicator...", required=False)
 
+    def get_choices(self, string, mytype, isString=False, use_regex=False):
```

Comment:
Method definitions should be placed after the special method `__init__`. Also, have you checked if you can use `_get_choices` method from `ChoicesMixin` instead of defining this method?